### PR TITLE
Add examples for model conversion

### PR DESCRIPTION
**Pull request by filipmoreau**
* PyTorch -> .onnx
* PyTorch -> .tflite
* TensorFlow -> .onnx
* TensorFlow -> .tflite
* .onnx -> cv25
* .onnx -> cv75

Change-Id: Iaabaf8146025dc4cd56faa28f298675cf2ce8e79

### CODE REVIEW

File: model-conversion/pytorch-to-onnx.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+*Copyright (C) 2026, Axis Communications AB, Lund, Sweden. All Rights Reserved.*
+
+# PyTorch model -> .tflite
```
Comment (FilipAlberius-axis) [inline model-conversion/pytorch-to-onnx.md]:
```suggestion
# PyTorch model -> .onnx
```

**Review by FilipAlberius-axis — APPROVED** [review]
